code/modules/mob/living/carbon/alien/life.dm

```diff
@@ -63,4 +63,7 @@
 
 /mob/living/carbon/alien/CheckStamina()
 	setStaminaLoss(max((staminaloss - 2), 0))
+	return
+
+/mob/living/carbon/alien/handle_changeling()

```

Comment:
Without this a runtime would of occured due to changeling stuff not existing for aliens